ui/src/components/policy/PolicyMonacoEditor.css

```diff
@@ -0,0 +1,21 @@
+.infoError {
```

Comment:
@saillinux  to review - Any possibility to avoid the CSS here?

Reply:
We will need to take this approach in terms of UI as this implementation will use too much real estate to show issues in the policy. Let's try show these messages in the editor instead of using the message component.

Reply:
Okay, I will remove the message components.
Do I only keep the display on hover or do you want me to add another source of information than on hover?

Do you want me to remove the entire css? Considering that the `warningIcon` class is used to display the warning indicator in the margin.

Reply:
Thanks for this PR btw, this is an awesome change!

Here are my thoughts:

1) I think we should perform the lint check automatically, and remove the Check button

2) If possible, we should display the issues in a hover (Similar to a JSON lint error, perhaps a different color so the user knows it is a different class of error).

3) A minimal amount of CSS is fine if required, we just try to keep it low to make frontend development easier and maintainable.

We can revisit later if users find that viewing the lint errors in the editor is not sufficient. One possibility: 1) Conditionally showing [Accordion](https://react.semantic-ui.com/modules/accordion/) below editor if there are linting issues. User has the option to expand it and see issues in detail, or 2) Show lint errors on the separate Issues page. Though this could be too far removed  from the policy, and confusing.

Reply:
Looks great! 

I think the Assume Role Policy Document needs to be treated differently, or avoided?

![image](https://user-images.githubusercontent.com/7538233/110367456-88e30e80-7ffc-11eb-990e-b959ad4e8b3b.png)

Reply:
> Looks great!
> 
> I think the Assume Role Policy Document needs to be treated differently, or avoided?
> 

What are your suggestions on this? What is the behavior a user would want here?
I can avoid the Role Policy Document for now if you think it's better.

Reply:
Yes I think we should avoid it for now, I couldn't find any references in the Parliament codebase that indicated it will analyze these policies.

Reply:
I've added an optional `enableLinting` props to the `PolicyMonacoEditor` component that allows to control whether or not we want policy linting on the editor instance.